app/components/EditProfileForm.tsx

```diff
@@ -49,6 +52,8 @@ export default function EditProfileForm({userData}) {
 
     const data = await res.json();
     console.log(data)
+
+    router.push("/dashboard")
   }
```

Comment:
Only redirect to dashboard on successful submission, otherwise display the error.